types/bulma-calendar/index.d.ts

```diff
@@ -0,0 +1,472 @@
+// Type definitions for bulma-calendar 6.0
+// Project: https://creativebulma.net/components/calendar
```

Comment:
Can this be replaced with:
https://github.com/Wikiki/bulma-calendar
? The url from `package.json` seems to be dead (404)

Reply:
I used https://creativebulma.net/product/calendar/demo, because this contains the documentation.

Reply:
that's ok. Can we swithc to that one? The url from package seems to be dead one.